src/index.html

```diff
@@ -7,8 +7,90 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="/" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="Moyo-Logo"
+          class="moyo__icon"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__items">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__items">
```

Comment:
```suggestion
        <img
          src="./images/logo.png"
          alt="Moyo-Logo"
          class="moyo__icon"
        >
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__items">
            <a
              class="nav__link is-active"
              href="#"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__items">
```
fix your code style according to the checklist